<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Rui Maciel</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Femp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionReopen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionNewModelWindow"/>
     <addaction name="actionNewPostprocessingWindow"/>
     <addaction name="actionNewAnalysisResultsWindow"/>
     <addaction name="actionNewFemEquationWindow"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="separator"/>
    <addaction name="actionWindowTile"/>
    <addaction name="actionWindowCascade"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewSelection"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Project</string>
    </property>
    <widget class="QMenu" name="menuDump">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Dump</string>
     </property>
     <addaction name="actionDump_FEM_equation"/>
     <addaction name="actionResults_from_selection"/>
    </widget>
    <addaction name="actionRun"/>
    <addaction name="menuDump"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionNodeRestraints"/>
    <addaction name="actionNodeActions"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionEditMaterials"/>
    <addaction name="actionQuadrature_rules"/>
    <addaction name="actionSelection"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuProject"/>
   <addaction name="menuView"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCommand_line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Command line</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionImportMesh">
   <property name="text">
    <string>Import mesh</string>
   </property>
  </action>
  <action name="actionTogglePerspective">
   <property name="text">
    <string>Toggle perspective</string>
   </property>
  </action>
  <action name="actionNodeRestraints">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Node restraints</string>
   </property>
  </action>
  <action name="actionNodeActions">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Node actions</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionShowNodalDisplacements">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nodal displacements</string>
   </property>
  </action>
  <action name="actionNodal">
   <property name="text">
    <string>Nodal</string>
   </property>
  </action>
  <action name="actionShowNodalForces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nodal forces</string>
   </property>
  </action>
  <action name="actionShowDomainForces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Domain forces</string>
   </property>
  </action>
  <action name="actionShowSurfaceForces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Surface forces</string>
   </property>
  </action>
  <action name="actionViewActions">
   <property name="text">
    <string>Actions</string>
   </property>
  </action>
  <action name="actionDisplaySurfaces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Faces</string>
   </property>
  </action>
  <action name="actionDisplayNodes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nodes</string>
   </property>
  </action>
  <action name="actionEditMaterials">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Materials</string>
   </property>
  </action>
  <action name="actionWindowTile">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="actionWindowCascade">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionViewModel">
   <property name="text">
    <string>Model</string>
   </property>
  </action>
  <action name="actionViewDisplacements">
   <property name="text">
    <string>Displacements</string>
   </property>
  </action>
  <action name="actionViewPostprocessing">
   <property name="text">
    <string>Postprocessing</string>
   </property>
  </action>
  <action name="actionViewStrains">
   <property name="text">
    <string>Strains</string>
   </property>
  </action>
  <action name="actionQuadrature_rules">
   <property name="text">
    <string>Quadrature rules</string>
   </property>
  </action>
  <action name="actionDump_FEM_equation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>FEM equation</string>
   </property>
  </action>
  <action name="actionReopen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reopen</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionSelection">
   <property name="text">
    <string>Selection</string>
   </property>
  </action>
  <action name="actionViewSelection">
   <property name="text">
    <string>Selection</string>
   </property>
  </action>
  <action name="actionSetModelViewport">
   <property name="text">
    <string>Model</string>
   </property>
  </action>
  <action name="actionSetPostprocessingViewport">
   <property name="text">
    <string>Postprocessing</string>
   </property>
  </action>
  <action name="actionResults_from_selection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Results from selection</string>
   </property>
  </action>
  <action name="actionNewModelWindow">
   <property name="text">
    <string>Model</string>
   </property>
  </action>
  <action name="actionNewPostprocessingWindow">
   <property name="text">
    <string>Viewport</string>
   </property>
  </action>
  <action name="actionNewAnalysisResultsWindow">
   <property name="text">
    <string>Analysis results</string>
   </property>
  </action>
  <action name="actionNewFemEquationWindow">
   <property name="text">
    <string>FEM equation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
